src/Main/MonkSpreadsheetTab.js

```diff
@@ -0,0 +1,131 @@
+    const getAbility = spellId => abilityTracker.getAbility(spellId);
+    const SGability = getAbility(SPELLS.SHEILUNS_GIFT.id);
+    const SGcasts = SGability.casts || 0;
+
+    const styles = {
+      cellBorder: { borderTop: '.5px solid #dddddd' },
+      table:  {borderBottom: '1px solid #dddddd', borderTop: '1px solid #dddddd', align: 'left', padding: '8px', float: 'left', margin: '2px' },
+    };
+
+    console.log('Log Tab: ', parser);
+    return (
+      <div>
+        <div className="panel-heading">
+          <h2>Player Log Data</h2>
+        </div>
+        <div style={{ padding: '15px 22px 15px 15px' }}>
+          <div style={{ padding: '0px 22px 15px 0px' }}>Please use the below table to populate the Player Log section of the Mistweaver Spreadsheet by Garg.  <a href="http://www.peakofserenity.com/mistweaver/spreadsheet/" target="_blank" rel="noopener noreferrer">Link to the sheet</a><br /></div>
+          <div>
+          <table style={styles.table}>
+            <tr><td>Fight Length (Minutes)</td></tr>
+            <tr><td>Fight Length (Seconds)</td></tr>
+            <tr style={styles.cellBorder}><td>Mana Remaining</td></tr>
+            <tr style={styles.cellBorder}><td>Absorbless DTPS</td></tr>
+            <tr style={styles.cellBorder}><td>Thunder Focus Tea Casts</td></tr>
+            <tr><td>Effuse</td></tr>
+            <tr><td>Enveloping Mist</td></tr>
+            <tr><td>Essence Font</td></tr>
+            <tr><td>Renewing Mist</td></tr>
+            <tr><td>Vivify</td></tr>
+            <tr style={styles.cellBorder}><td>Total Uplifting Trance procs</td></tr>
+            <tr><td>Unused UT %</td></tr>
+            <tr style={styles.cellBorder}><td>Mana Tea MP5</td></tr>
+            <tr style={styles.cellBorder}><td>Lifecycles-EnM</td></tr>
+            <tr><td>Lifecycles-Vivify</td></tr>
+            <tr style={styles.cellBorder}><td>SotC Mana Return</td></tr>
+            <tr style={styles.cellBorder}><td>Average SG Stacks</td></tr>
+            <tr style={styles.cellBorder}><td>Effective WoS %</td></tr>
+            <tr style={styles.cellBorder}><td>Targets per Celestial Breath</td></tr>
+            <tr style={styles.cellBorder}><td>The Mists of Sheilun Procs</td></tr>
+            <tr><td>Targets per TMoS</td></tr>
+            <tr style={styles.cellBorder}><td>Effective RJW %</td></tr>
+            <tr style={styles.cellBorder}><td>Dancing Mist Healing</td></tr>
+            <tr style={styles.cellBorder}><td>Mastery per EF</td></tr>
+            <tr style={styles.cellBorder}><td>Targets per Essence Font</td></tr>
+            <tr><td>Targets per Chi Burst</td></tr>
+            <tr style={styles.cellBorder}><td>Misc MP5</td></tr>
+            <tr><td>Misc HPS</td></tr>
+            <tr style={styles.cellBorder}><td>Tier 20 2 Piece MP5</td></tr>
+            <tr style={styles.cellBorder}><td>Tier 20 4 Piece Uptime</td></tr>
+          </table>
+          <table style={styles.table}>
```

Comment:
The initial reason I split them into two tables is to make the copy and paste from the table into the spreadsheet easier for the user.  If this is all in a single table, the user would need to also overwrite row headings.  I understand the through about this potentially being error prone - but I wanted to make it easier for a user to use even if it potentially makes the maintenance more difficult.